notebooks/223-text-prediction/223-text-prediction.ipynb

```diff
@@ -30,7 +30,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        ov_model = ov.convert_model(onnx_path, input="input_ids[1,-1],attention_mask[1,-1]")
</pre><p>please use tuples instead of specification input as string</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>done</p>